app/assets/javascripts/Measure.js

```diff
@@ -52,6 +52,7 @@ const MeasureSchema = mongoose.Schema(
     populations: [Mixed],
     populations_cql_map: Mixed,
     observations: [Mixed],
+    population_ids: Mixed,
```

Comment:
What would an example of this look like? And what would use this?

Reply:
I believe this would look like:
```
"population_ids": {
    "IPP": "8CBC9E66-9B3D-42A0-8314-8E5004687AA0",
    "MSRPOPL": "ED6A010A-76E8-491A-8095-2F42F2F3AEF5",
    "OBSERV": "385D1BA9-E97D-4473-81BC-B96F0AFDE2D0",
    "MSRPOPLEX": "E1CF58C6-117B-4BAD-B02F-B0740AA244F8"
  }
```

This was added to the measure fixtures in `js-ecqm-engine`; I believe it was needed for cql calculation.

Reply:
But if there are multiple "population sets" (Example: Stratification 1, Strat 2...) how would this look? I might be missing something...

Reply:
Ah, so this isn't a Measure object like we would think of it in Ruby. This is a specific population of a measure? Should the name be changed then?